test/test.cpp

```diff
@@ -0,0 +1,88 @@
+#include <loudnessIO/SoundFile.hpp>
+#include <loudnessIO/ProcessFile.hpp>
+#include <loudnessAnalyser/LoudnessAnalyser.hpp>
+
+#include "gtest/gtest.h"
+
+#include <string>
+#include <cmath>
+#include <stdlib.h>
+#include <iostream>
+
+// Value of EBU_LOUDNESS_TEST_SET_PATH environment variable
+std::string ebuLoudnessTestSetPath;
+
+/**
+ * @brief Callback used while processing, to print progression of the loudness analysis.
+ */
+void progress(const int p)
+{
+    std::cout << "[" << p << "%]\r" << std::flush;
```

Comment:
Agreed, but I think it can be interesting to add this small check, and prefers to display anything to don't have long messages in unit logs... (in logs, progress are printed with a break line on each update)